modules/applications/examples/ui-builder/production/docker-compose.yml

```diff
@@ -32,8 +31,4 @@ services:
     volumes:
       - <YOUR_LICENSE_FOLDER>:/opt/bonita_lic
     environment:
-      - <ANY_STANDARD_BONITA_ENV_VARIABLE_HERE>
-    ports:
```

Comment:
**suggestion:** I don't understand why we should remove the expose ports. It will not working ?

Reply:
It's possible to keep it but; Bonita is already accessible via the proxy using /bonita path